### PR TITLE
Js testing

### PR DESCRIPTION
**Pull request by adambrown13**
I added a node-module/mock because they require specific configuration. But the rest of the node-module is built on npm installation of karma. Should I change the ReadMe to reflect that?

### CODE REVIEW

**Comment by shankari** [comment]
Can you highlight the changes that you made to `node_modules/angular_mocks`?

The README says that it is intended to be installed via npm or bower, so I just want to understand why it needs to be checked in here.

**Comment by adambrown13** [comment]
ReadMe updated and Angular-mocks removed

File: README.md [hunk 1]
```diff
@@ -41,6 +41,17 @@ Installation is now complete. You can view the current state of the application
...
+JS Testing
+---
+From the root directory run
+    $ npm install karma --save-dev
+    $ npm install angular-mocks
+    $ npm install karma-jasmine karma-chrome-launcher --save-dev
+
+Write tests in www/js/test
+To run tests, run 
+    $ karma start my.conf.js in the root directory
```
Comment (shankari) [inline README.md]: similar comments as the server side wrt `karma` path

**Comment by shankari** [comment]
Getting a similar failure as the server side.

```
21 06 2016 13:00:41.239:INFO [karma]: Karma v0.13.22 server started at http://localhost:8080/
21 06 2016 13:01:03.928:INFO [Firefox 47.0.0 (Mac OS X 10.11.0)]: Connected on socket /#GZcA0ZSWv8xOmyqnAAAA with id manual-6122
Firefox 47.0.0 (Mac OS X 10.11.0) LOG: 'WARNING: Tried to load angular more than once.'

Firefox 47.0.0 (Mac OS X 10.11.0) Testing Functions that are declared, but have no code in controller.js RootCtrl takes a scope variable FAILED
        TypeError: angular.element.cleanData is not a function in /private/tmp/e-mission-phone/node_modules/angular-mocks/angular-mocks.js (line 2915)
        module.$$cleanup@/private/tmp/e-mission-phone/node_modules/angular-mocks/angular-mocks.js:2915:7
        module.$$afterEach@/private/tmp/e-mission-phone/node_modules/angular-mocks/angular-mocks.js:2885:7
```

**Comment by adambrown13** [comment]
Similar changes as those made on the server side

**Pull request merged**